src/components/toJSONStream.js

```diff
@@ -0,0 +1,15 @@
+const parallelTransform = require('parallel-transform');
+const logger = require( 'pelias-logger' ).get( 'whosonfirst' );
+
+const maxInFlight = 10;
+
+module.exports.create = () => {
+  return parallelTransform(maxInFlight, function(record, next) {
+    try {
+      next(null, JSON.parse(record.body));
+    } catch (parse_err) {
+      logger.error(`exception parsing JSON for id ${record.id}: ${parse_err}`);
+      next(parse_err);
```

Comment:
This will cause the whole importer to fail at this point and the process to `exit(1)`, is this your intention?

It might be preferable to logging a warning if one record is malformed rather than aborting the whole import.

Reply:
This was not my intention, thx again :smile: